spec/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/vm_spec.rb

```diff
@@ -86,4 +86,36 @@
       expect(vm.power_state).to eq('on')
     end
   end
+
+  describe '#raw_destroy' do
+    before(:each) do
+      allow(vm).to receive(:provider_object).and_return(instance)
+    end
+
+    let(:parent) do
+      vpc = double("ManageIQ::Providers::IbmCloud::CloudTools::Vpc")
+      allow(vpc).to receive(:logger).and_return(Logger.new(nil))
+      allow(vpc).to receive_messages(:cloudtools => nil, :region => nil, :version => nil, :request => nil)
+      vpc
+    end
+
+    let(:actions) do
+      actions = ManageIQ::Providers::IbmCloud::CloudTools::VpcSdk::InstanceActions.new(:vpc => parent, :instance_id => nil)
+      allow(actions).to receive(:create).and_return({:this => 'mock'})
+      actions
+    end
+
+    let(:instance) do
+      instance = ManageIQ::Providers::IbmCloud::CloudTools::VpcSdk::Instance.new(:vpc => parent, :data => {})
+      allow(instance).to receive(:refresh) { instance.merge!({:id => 'mock_id', :name => 'Test instance', :status => 'running'}) }
+      allow(instance).to receive(:actions).and_return(actions)
```

Comment:
Since you aren't calling `instance.actions` you don't need this line or [this block](https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/217/files#diff-354337d705a3f216fbf4a6300b4ee15678f48f80cbb8a10ddd136977855cb7dfR102-R106)